### ISSUE TITLE
Implement GitHub Actions

### ISSUE THREAD

**Issue by dave-burke** [skill-collectors/agile-poker#9]
### On the back end
  - lint
  - test
  - deploy to "QA"

### On the front end
  - check
  - lint
  - test:unit
  - test:e2e
  - build
  - deploy

Ideally check/test/build the front/backend in parallel, then deploy backend then frontend when both are good.

Can we have separate deploys per PR?

## When a PR is merged into `main`

Do the same thing as with PRs, except deploy to "prod"

Workflows in https://github.com/dave-burke/rpn-calc/ may be a good starting point.

## Don't forget

- Update the documentation to note the URL where people can check QA. Consider putting the URL in the pull request template.
